.github/workflows/release-crypto-icons.yml

```
@@ -0,0 +1,44 @@
+name: "[Release] crypto icons"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs at midnight UTC every Monday
+    - cron: "0 0 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-crypto-icons:
+    environment: ${{ github.event.inputs.environment }}
```

This will not work. You need to set the environment directly. There is no input for the schedule trigger or workflow dispatch. i will create environment `production-icons`

```suggestion
    environment: production-icons
```

---

fixed in https://github.com/trezor/trezor-suite/pull/12694/commits/8489b3c8d033ea65d640e0e8d9a361864c24ab55